css-three-column-layout/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Three Column Layout</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <h2>
+          Designed for life science. Optimized for SaMD and Digital Health.
+        </h2>
+      </div>
+      <div class="row">
+        <div class="column">
+          <div class="circle-container">
+            <img src="./images/merge.png" alt="" />
```

Comment:
You have this `column` class but I want to see more descriptive classes for layout. This is what I would expect this `column` class to do:
```
.column {
  display: flex;
  flex-direction: column;
}
```
What about the size of the column? Or anything else about it? Will elaborate down in the CSS.